cachito/web/validation.py

```diff
@@ -94,3 +95,17 @@ def handle_wrapper(request):
                 raise ValidationError(exception.detail)
 
         return handle_wrapper
+
+
+@jsonschema.draft4_format_checker.checks("date")
+def date_validator(val):
+    """Validate that date-type fields have the correct format."""
+    if not isinstance(val, str):
+        return False
+
+    try:
+        datetime.datetime.fromisoformat(val)
+    except ValueError:
+        raise ValidationError(f"'{val}' is not a valid datetime.")
```

Comment:
for my information, what is the difference between returning False and raising ValidationError?

Reply:
Returning False raises the default Connexion exception, which we don't want. I'll change the code so that only the Cachito exception is raised. Thanks, Adam!